lib/bank_account/saving.rb

```diff
@@ -0,0 +1,45 @@
+module BankAccount
+	
+
+	class Saving < Account
+
+		def initialize(id, initial_balance)
+			super	
+			if @initial_balance < 10 
+				raise ArgumentError, "an error has occurred. Balance must equal more than $10. Please make a deposit."
+			end	
+			@available_balance
```

Comment:
@available_balance is not being set to anything here. If you don't need it later, you can just delete line 11. Otherwise you will want to set it to something.